extensionsGallery-insider.json

```diff
@@ -909,63 +909,63 @@
 									"value": "*"
 								}
 							]
-						},
+						}
+					],
+					"statistics": [],
+					"flags": "preview"
+				},
+				{
+					"extensionId": "27",
```

Comment:
for insider build, the new DB extension was added to the wrong place, as a result I am not seeing it in my insiders build ADS.